src/ObjCRuntime/PlatformAvailability2.cs

```diff
@@ -205,4 +205,83 @@ public sealed class UnavailableAttribute : AvailabilityBaseAttribute
 		{
 		}
 	}
+
+	public sealed class TVAttribute : IntroducedAttribute
+	{
+		public TVAttribute (byte major, byte minor)
+			: base (PlatformName.TvOS, (int)major, (int)minor)
+		{
+		}
+
+		public TVAttribute (byte major, byte minor, bool onlyOn64 = false)
```

Comment:
What's the point of default onlyOn64 parameter `false` value ? I cannot think of case where it will be used

Reply:
I just copy pasted the existing API from PlatformAvailability.cs to keep the API. I could have searched for usage and removed some but:

- I'd also have to search all branches in flight, such as any release/xcode not merged to master
- You lose some symmetry if you can do [Mac (10, 12, onlyOn64 : true) but not [TV (11, 0, onlyOn64 : true)]

Reply:
My comment is not about API but about default false value for onlyOn64, why is it there?

Reply:
I'm not sure what you are asking. 

Technically we are matching the original platform availability API found here:

https://github.com/xamarin/xamarin-macios/blob/master/src/ObjCRuntime/PlatformAvailability.cs#L593

In general, we have that override to allow us to mark certain APIs as 64-bit only as Apple often loves to either remove 32-bit implementations or never provide them.

If you are asking why we have it = false when that gives it the same behavior as above we want:

[TV (11, 0)] to not be 64-bit only and if it was = true we'd get that. The common pattern is [TV (11, 0, onlyOn64 : true)] to make it more clear in bindings what the true is for.

Looking at it, we maybe could have collapsed it to one ctor with more default parameters like

(byte major, byte minor, byte point = 0, bool onlyOn64 = false)

But I was looking to match the old API as close as possible.

Also, the reason that we never needed this was that these were processed by PMCS and converted into other attributes before the compiler ever saw them (see deleted code).

Reply:
> If you are asking why we have it = false when that gives it the same behavior as above we want:

Yes as `[TV (11, 0)]` won't use this overload, the only way to use it is to use explicit parameter name but then the default value won't be used

Reply:
tvOS is always 64-bits, so the `onlyOn64` parameter is redundant for tvOS.

The opposite logic goes for watchOS (it's always 32-bits), but I guess that can change in the future, so the parameter is fine.